README.md

```diff
@@ -35,6 +35,7 @@ We currently have 27 recipes available. For recipes in development, check out [t
 
 ### Entrees
 
+- [Nasi goreng](entrees/nasi-goreng.md)
```

Comment:
Can you run the python script to update this document instead please?

Reply:
Done